src/OrchardCore.Modules/OrchardCore.Users/Views/Items/CustomUserSettingsDeploymentStep.Fields.Summary.cshtml

```diff
@@ -0,0 +1,24 @@
+@model ShapeViewModel<CustomUserSettingsDeploymentStep>
+
+@{
+    var includeAll = Model.Value.IncludeAll;
```

Comment:
Why is this needed?

Reply:
It similar to `CustomSettingsDeploymentStep.Fields.Summart.cshtml`